lib/cldr/export/data/country_codes.rb

```diff
@@ -0,0 +1,28 @@
+module Cldr
+  module Export
+    module Data
+      class CountryCodes < Base
+        def initialize
+          super(nil)
+          update(:CountryCodes => country_codes)
+        end
+
+        private
+
+        def country_codes 
+          doc.xpath("//codeMappings/*").each_with_object({}) do |node, hash|
+            if (node.name == "territoryCodes")
+              hash[node[:type]] = {}
+              hash[node[:type]]["three_digit_code"] = node[:numeric] if node[:numeric]
+              hash[node[:type]]["three_letter_code"] = node[:alpha3] if node[:alpha3]
```

Comment:
```suggestion
              type = node.attribute('type').to_s.to_sym
              hash[type] = {}
              hash[type]["three_digit_code"] = node[:numeric] if node[:numeric]
              hash[type]["three_letter_code"] = node[:alpha3] if node[:alpha3]
```
I would suggest using `.attribute('type')` for consistency with other data parsers.